src/twisted/web/test/test_http.py

```diff
@@ -1469,6 +1469,21 @@ def finished(extra):
         self.assertEqual(errors, [])
         self.assertEqual(successes, [True])
 
+    def test_tailerServerTiming(self):
+        """
+        L{_ChunkedTransferDecoder.dataReceived} decodes chunked-encoded data
+        and correctly handle trailer Server-Timing data.
```

Comment:
Can you please add more details about what correctly means?

We alwasy write tests to make sure the implementation is correct :) 

So it helps to skip the "correctly handles" part and add a few more details about the scope / purpose / expected behaviour in this test.